apps/actors/lib/actors/actor/entity/entity.ex

```diff
@@ -533,6 +600,39 @@ defmodule Actors.Actor.Entity do
        ),
        do: timeout + timeout_factor
 
+  defp handle_schedulers(schedulers) when is_list(schedulers) do
+    schedulers
+    |> Flow.from_enumerable()
+    |> Flow.map(fn %ScheduledTimerCommand{expression: expression, command: command} = cron_command ->
+      Logger.debug("Registering cron command #{inspect(cron_command)}")
+      delay = get_time_from_cron(expression)
+      Process.send_after(self(), {:invoke_timer_command, command}, delay)
+    end)
+    |> Flow.run()
+  end
+
+  defp handle_schedulers(nil), do: :ok
+
+  defp handle_schedulers([]), do: :ok
+
+  defp handle_timers(timers) when is_list(timers) do
+    timers
+    |> Flow.from_enumerable()
+    |> Flow.map(fn %FixedTimerCommand{seconds: delay, command: command} = timer_command ->
+      Logger.debug("Registering timer command #{inspect(timer_command)}")
+      Process.send_after(self(), {:invoke_timer_command, command}, delay)
+    end)
+    |> Flow.run()
+  end
+
+  defp handle_timers(nil), do: :ok
+
+  defp handle_timers([]), do: :ok
+
+  defp get_time_from_cron(_expression) do
+    10000
```

Comment:
idk if having the dependency of Crontab here would help

Reply:
Yeah! I was looking for some lib that does this